src/drive/mobile/containers/onboarding/SelectServer.jsx

```diff
@@ -9,10 +9,36 @@ import { registerDevice, setUrl } from '../../actions/settings'
 import styles from '../../styles/onboarding'
 
 export class SelectServer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      fetching: false
+    }
+  }
+
+  async setServerUrl(serverUrl) {
+    this.setState({ fetching: true })
+    try {
+      await this.props.registerDevice(serverUrl)
+      this.props.nextStep()
+    } catch (e) {
+      console.error(e)
+    } finally {
+      this.setState({ fetching: false })
+    }
+  }
+
   componentDidMount() {
     this.serverInput.focus()
   }
 
+  componentDidUpdate() {
+    if (this.props.error) {
+      this.serverInput.focus()
+      this.serverInput.select()
```

Comment:
What does `.select()` do?

Reply:
It selects the text in the input. Unless I was mislead, `.focus()` just put the cursor inside the field, ready to write and `select()` selects the string, ready to delete. 
I was asked to add this feature. :)

Reply:
Cool, makes sense then!